doc.go

```diff
@@ -1,10 +1,10 @@
 /*
 Package fx is the UberFx Service Framework.
 
-[][doc]
-[][cov]
-[][ci]
-[]report-card (https://goreportcard.com/report/github.com/uber-go/fx)
+GoDoc (https://godoc.org/go.uber.org/fx)
```

Comment:
@glibsm I finally fixed this stuff (or more accurately, the v2 api of the markdown parser I'm using finally recognizes these).